gulp/index.js

```diff
@@ -305,9 +305,9 @@ module.exports = {
      * Removes all build artifacts.
      */
     gulp.task('clean', function(cb) {
-      gutil.log(util.format('Cleaning: %s', gutil.colors.magenta(paths.build)));
-      var targets = [ paths.build ];
-      return del(targets, { force: true });
+      var target = paths.build + '/**/*';
```

Comment:
Wait, users won't do that, as they don't want to set `build` to a glob pattern :).   This seems fine.  We might want to eventually expose a `paths.clean` key, but we don't need to do that now.